src/print.js

```diff
@@ -1,3 +1,37 @@
-export default function printMe() {
-    console.log('I get called from print.js!');
-  }
\ No newline at end of file
+export default class PrintMe {
+  constructor() {
+    this.addedTasks = [
+      {
+        id: 1,
+        completed: false,
+        description: 'Setup webpack',
+      },
+      {
+        id: 1,
+        completed: false,
+        description: 'Set up webpack dev-server',
+      },
+      {
+        id: 1,
```

Comment:
- [ ] Kindly make sure to change the indexes of the second and third tasks, it should be `2` then `3`